public/js/graph/sidebar/sidebar_divs.js

```diff
@@ -89,8 +89,13 @@ function toggleSidebar(location) {
             $('#sidebar-icon').addClass('flip');
         });
         fillFCECount();
-        $('#focuses').show();
-        $('#focuses-nav').addClass('active');
+
+        $('#graphs').empty();
```

Comment:
It doesn't actually need to be that way, thinking about it now. I changed it so that the graph buttons are created just once on page load.